### ISSUE TITLE
interface setting always empty

### ISSUE THREAD

**Issue by c-o-m-m-a-n-d-e-r** [lvogt/ioBroker.wireless-mbus#133]
**Describe the bug**  
when open the instance for edit the interface entry is always empty

**To Reproduce**  
just edit the installed instance. everything is setup and also working, but after revisiting the instance settings, the interface /dev/... is empty (but still works when i dosen't change anything

**Expected behavior**  
the configured interface should apear as path

**Screenshots & Logfiles**  
If applicable, add screenshots and logfiles to help explain your problem.

**Versions:**  
 - Adapter version: 0.9.0
 - JS-Controller version: 4.0.24
 - Node version: 18.16.0
 - Operating system: Docker (on Synology Host)

**Additional context**  
![Bildschirmfoto 2023-06-02 um 10 06 36](https://github.com/lvogt/ioBroker.wireless-mbus/assets/815885/b7e46d61-ca10-47bb-a802-2bda5bfaee35)

**Comment by lvogt** [comment]
Could please add a debug log for when you open the admin page?
The field should get filled in correctly after the admin page requests the current list of serial ports from ioBroker - maybe something is broken there.

**Comment by c-o-m-m-a-n-d-e-r** [comment]
of course ... can deliver on monday ... atm at the train ... tomorrow late back home.

thanks for looking in!

**Comment by c-o-m-m-a-n-d-e-r** [comment]
i'm little bit confused ... is there any information i can filter at ?
switched to debug but only see message debugs from my meters... nothing from the interface itself ... or is there something special to do to get admin interface debug log ?


thanks

**Comment by lvogt** [comment]
There should be a few messages when actually open the admin interface of the adapter because it queries some information from it


Am 5. Juni 2023 11:09:18 MESZ schrieb Paul ***@***.***>:

**Comment by c-o-m-m-a-n-d-e-r** [comment]
hi,
i double checked, the only thing i found is this
`

2023-06-07 07:21:11.829 - info: admin.0 (1908) ==> Connected system.user.admin from ::ffff:192.168.10.21
--
2023-06-07 07:21:12.230 - info: wireless-mbus.0 (17058) List of port: []
2023-06-07 07:21:15.264 - info: wireless-mbus.0 (17058) List of port: []
2023-06-07 07:21:18.309 - info: wireless-mbus.0 (17058) List of port: []
2023-06-07 07:21:18.807 - info: admin.0 (1908) <== Disconnect system.user.admin from ::ffff:192.168.10.21

`

all the other debug stuff is message receiving and decrypting etc.

**Comment by lvogt** [comment]
Interesting. Do you use the new "serial over tcp socket" feature? Or what do you have configured as your serial port?

**Comment by c-o-m-m-a-n-d-e-r** [comment]
no, i only mapped that device via : /dev/im871a:/dev/im871a and also in the ENV with : USBDEVICES=/dev/im871a
/dev/im871a is just via serial mapped to /dev/ttyUSB0

i can test from master branch ... gimme a second

**Comment by c-o-m-m-a-n-d-e-r** [comment]
I can report success ... with 0.9.1 it is working as expected.

Thanks a lot!

**Comment by lvogt** [comment]
Thanks for reporting and testing :)
I published 0.9.1 to npm now as well.
